docs/setupandinstallation.md

```diff
@@ -134,29 +134,58 @@ To configure the storefront, use the `withConfig` method on the StorefrontModule
   ],
 ```
 
-You do not have to specify a config if you do not need to override its value. For example, if you only need to override the back end base URL, you can use this config:
+You do not have to specify a config if you do not need to override its value. For example, if you only need to override the back end base URL, you can use the following config:
 
-```
+```typescript
 imports: [BrowserModule, StorefrontModule.withConfig({
   server: {
     baseUrl: 'https://my-custom-backend-url:8080',
   }
 })]
 ```
 
+## Configuring the Base URL
+
+You can configure the base URL with a special html `meta` tag, instead of hard coding it in the `withConfig` method of the StorefrontModule. This allows you to deploy to different environments with only one compiled JavaScript application, because you only need to modify the `meta` tag of the `index.html` file for each environment.
+
+The following example shows how the `meta` tag can be configured in the `index.html` file:
+
+```html
+<meta name="occ-backend-base-url" content="https://my-custom-backend-url:8080" />
+```
+The corresponding `app.module.ts` file appears as follows:
+
+```typescript
+  imports: [
+    BrowserModule, StorefrontModule.withConfig({
```

Comment:
@Platonn for this example I'd leave out the config as a whole, or at least remove non-related pieces such as authentication